lib/y2lan_restart_common.pm

```diff
@@ -47,7 +47,7 @@ sub initialize_y2lan {
 sub open_network_settings {
     type_string "yast2 lan\n";
     accept_warning_network_manager_default;
-    assert_screen 'yast2_lan', 100;       # yast2 lan overview tab
+    assert_screen 'yast2_lan', 180;       # yast2 lan overview tab
```

Comment:
even this I also see unrelated to the commit message or PR description

Reply:
It was failing with a timeout on more than 10 % of the tests runs. Even with low system load (only one worker at a time) it often takes more than two minutes for yast2 lan to exit when there is more than one NIC, which is the case in the "extra tests for gnome" on sle 12.